pkg/ranchhand/run.go

```diff
@@ -11,11 +11,12 @@ import (
 const OutputDirectory = "ranchhand-output"
 
 type Config struct {
-	SSHUser    string
-	SSHPort    uint
-	SSHKeyPath string
-	Nodes      []string
-	Timeout    time.Duration
+	SSHUser         string
+	SSHPort         uint
+	SSHKeyPath      string
+	Nodes           []string
```

Comment:
I guess what might have done here is group is the public and private IPs into a single object

```go
type Node struct {
  PublicIP string
  PrivateIP string
}

type Config struct {
  Nodes []Node
}